Makefile

```diff
@@ -94,6 +98,9 @@ endif
 ### rollout: rebuild and push docker image and restart cluster deployment
 rollout: docker restart
 
+### configure: configures already deployed controller according to set env variables
+configure: _update_yamls _apply_controller_cfg _reset_yamls
```

Comment:
probably configure and update could be merged but not sure.
Maybe it makes sense to rename update to update_crds

Reply:
I'll do this in a followup PR.